.github/renovate.json5

```diff
@@ -298,17 +316,18 @@
       ],
       "allowedVersions": "<1.21",
       "matchBaseBranches": [
-        "v1.13"
+        "v1.14",
+        "v1.13",
+        "v1.12"
       ]
     },
     {
       "matchPackageNames": [
-        "docker.io/library/golang",
-        "go"
+        "docker.io/library/alpine"
       ],
-      "allowedVersions": "<1.21",
+      "allowedVersions": "<3.20",
       "matchBaseBranches": [
-        "v1.12"
+        "v1.15"
```

Comment:
The git diff here _looks_ like it's replacing the golang dep with the alpine, but this is a side effect of my previous comment. The code next to the previous comment moved the golang 1.20 dep into that rule, and this is a brand new rule to limit alpine updates for the 1.15 branch.